data/net/minecraft/util/Mth.mapping

```diff
@@ -30,10 +30,42 @@ CLASS net/minecraft/util/Mth
 		ARG 2 x
 	METHOD average ([J)D
 		ARG 0 values
+	METHOD binBiModalNormalDistribution (DDDDDDII)[D
+		ARG 0 firstPeakHeight
+		ARG 2 firstStandardDeviation
+		ARG 4 firstMean
+		ARG 6 secondPeakHeight
+		ARG 8 secondStandardDeviation
+		ARG 10 secondMean
+		ARG 12 lowerBound
+		ARG 13 upperBound
+	METHOD binLogDistribution (DDII)[D
+		ARG 0 slope
+		ARG 2 yIntercept
```

Comment:
This value affects the x intercept, but is not the x intercept itself.

Reply:
The parameter is `yIntercept`, not `xIntercept`

Reply:
That's literally impossible. The function called is `m * log(x) + b`, which x range is (0, infinity), so it cannot be the y intercept. `b` is typically defined as (when in slope-intercept form) the value of y when x = 0 hence why it gets inflated as the y intercept, but that cannot be true for a log function since x cannot equal 0 without an additional parameter inside the log parenthesis. The most it does is offset the y position by some amount.